testing/tests/DevExpress.ui.widgets/toolbar.tests.js

```diff
@@ -1445,6 +1464,10 @@ QUnit.module('adaptivity without hiding in menu', {
         };
 
         this.MEASURE_SAFE_TEXT = 'xyvxyv';
+        this.styles = $('<style>.dx-toolbar-label > div, .dx-toolbar-label .dx-toolbar-item-content > div { font-family: arial !important; }</style>').appendTo('head');
```

Comment:
Shouldn't it be better to separate this long row to several?

Reply:
done